ovn_cluster.sh

```diff
@@ -716,34 +716,35 @@ function build-images-with-ovn-rpms() {
 }
 
 function build-images-with-ovn-sources() {
+    if [ ! -d "${OVS_SRC_PATH}" ]; then
+        echo "Set the OVS_SRC_PATH var pointing to the location of ovs source code."
+        exit 1
+    fi
+
+    if [ ! -d "${OVN_SRC_PATH}" ]; then
+        echo "Set the OVN_SRC_PATH var pointing to the location of ovn source code."
+        exit 1
+    fi
+
+    echo "OVS_SRC_PATH = $OVS_SRC_PATH"
+    echo "OVN_SRC_PATH = $OVN_SRC_PATH"
+
     if [ ! -d ./ovs ]; then
-	    echo "OVS_SRC_PATH = $OVS_SRC_PATH"
-	    if [ "${OVS_SRC_PATH}" = "" ]; then
-            echo "Set the OVS_SRC_PATH var pointing to the location of ovs source code."
-            exit 1
-	    fi
-
-	    rm -rf ./ovs
-	    cp -rf $OVS_SRC_PATH ./ovs
-	    DO_RM_OVS='yes'
+        DO_RM_OVS='yes'
     fi
 
     if [ ! -d ./ovn ]; then
-	    echo "OVN_SRC_PATH = $OVN_SRC_PATH"
-	    if [ "${OVN_SRC_PATH}" = "" ]; then
-            echo "Set the OVN_SRC_PATH var pointing to the location of ovn source code."
-            exit 1
-	    fi
-	    rm -rf ovn
-	    cp -rf $OVN_SRC_PATH ovn
-	    DO_RM_OVN='yes'
+        DO_RM_OVN='yes'
     fi
 
+    cp -urf "${OVN_SRC_PATH}" ./ovn
```

Comment:
hm.... I was thinking about this some more.
If we are to do this, maybe we should get rid of these lines in Vagrantfile:
https://github.com/ovn-org/ovn-fake-multinode/blob/master/Vagrantfile#L45-L49
Could you please take care of that in this commit as well, please?